src/View/mainpage/WindowingSlider.py

```
@@ -266,19 +269,24 @@ def update_bar(self, index, top_bar=True):
         index = min(index, self.slider_density - 1)
 
         if top_bar:
-            self.slider_bars[self.top].setColor("white")
+            with suppress(RuntimeError):
+                self.slider_bars[self.top].setColor("white")
```

is there a better way than just suppressing errors? Also, this file is the for the windowing slider on the dicom view page right, what does this file have to do with image fusion views? I could be wrong but it seems odd we are adjusting this file.

---

i suppressed these at the start of the windowing, as it always threw errors and i couldnt test it. lets see if we dont need them now